server/.env.example

```diff
@@ -93,3 +98,29 @@ LOW_BATTERY_ALERT_TIMEOUT_TEST = 2
 # Login to our OneSignal Account. Get this value from Settings --> Messaging --> Android Categories --> Alerts and Reminders --> Channel ID
 ONESIGNAL_ALERT_ANDROID_CATEGORY_ID=guid-here
 ONESIGNAL_ALERT_ANDROID_CATEGORY_ID_TEST=android-group-guid
+
+# Name of the ClickUp Team where the Sensor Tracker lives
+# Get this from Clickup --> Settings --> Workspace Settings
```

Comment:
Might also want to note that this is also Brave's Team/Workspace, as that is the more the principle that this is used for in terms of security.

Reply:
I was just avoiding putting the full name. But I can make it a little clearer.

Reply:
Updated!